web/src/test/java/com/graphhopper/http/GraphHopperLandmarksIT.java

```diff
@@ -20,49 +20,77 @@
 import com.fasterxml.jackson.databind.JsonNode;
 import com.graphhopper.util.CmdArgs;
 import com.graphhopper.util.Helper;
-import org.junit.AfterClass;
 import org.junit.Before;
 import org.junit.Test;
 
 import java.io.File;
 
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.*;
 
 /**
  * Tests the creation of Landmarks and parsing the map.geo.json file
  *
  * @author Robin Boldt
  */
 public class GraphHopperLandmarksIT extends BaseServletTester {
-    private static final String DIR = "./target/north-bayreuth-gh/";
+    private static final String DIR = "./target/landmark-test-gh/";
 
-    @AfterClass
-    public static void cleanUp() {
+    @Before
+    public void setUp() {
         Helper.removeDir(new File(DIR));
-        shutdownJetty(true);
     }
 
-    @Before
-    public void setUp() {
+    @Test
+    public void testSimpleQuery() throws Exception {
```

Comment:
This query is really nothing special, we can remove it and save some time by only using your method, I don't think it adds more value than yours.